src/tools.rs

```diff
@@ -146,3 +146,5 @@ pub(crate) fn new_py_string<'py>(py: Python<'py>, s: &str, cache_str: StringCach
         pystring_fast_new(py, s, ascii_only)
     }
 }
+
+pub(crate) const UNION_ERR_SMALLVEC_CAPACITY: usize = 4;
```

Comment:
I might have just called this `SMALL_UNION_THRESHOLD`, but this is fine too 👍

Reply:
I'll rename in another PR - thinking of moving this to `common`, which we introduce in another one of the various union PRs :)

Reply:
See a04e297690f6f0834be6e840506a772e1599961d